client/src/components/MenuHeader.js

```diff
@@ -1,48 +1,109 @@
+import Login from '../components/Login';
+import FourOFour from '../components/404';
+import ScoreSheet from '../components/ScoreSheet';
+import Rules from '../components/Rules';
+import ProfileForm from '../components/ProfileForm';
+import HeadToHead from '../components/HeadToHead';
 
 class MenuHeader extends Component {
-  state = { activeItem: 'score form' }
+  
+  state = { 
+  	activeItem: 'score form',
+  	visible: false,
+  	dimmed: false 
+  }
+
+  toggleVisibility = () => this.setState({ visible:!this.state.visible, dimmed: !this.state.dimmed})
 
   handleItemClick = (e, { name, to }) => {
 		this.props.history.push(to);
 		this.setState({ activeItem: name });
 	}
 
   render () {
-  	const { activeItem } = this.state
+  	const { activeItem, visible, dimmed } = this.state
   	return (
-		  <Menu color='blue' inverted pointing secondary>
-		    <Menu.Item name='Fantasy Fitness'/>
-		    <Menu.Menu position='right'>
-		    	<Menu.Item to='/score' name='score form' active={activeItem === 'score form'} onClick={this.handleItemClick}>
-						Score
-					</Menu.Item>
-		    	<Menu.Item to='/scoresheet' name='score sheet' active={activeItem === 'score sheet'} onClick={this.handleItemClick}>
-						Score Sheet
-					</Menu.Item>
-		    	<Menu.Item to='/league' name='league' active={activeItem === 'league'} onClick={this.handleItemClick}>
-						League
-					</Menu.Item>
-					<Menu.Item to='/matchups' name='matchups' active={activeItem === 'matchups'} onClick={this.handleItemClick}>
-						Matchups
-					</Menu.Item>
-		    	<Menu.Item to='/rules' name='rules' active={activeItem === 'rules'} onClick={this.handleItemClick}>
-						Rules
-					</Menu.Item>					
-		    	<Menu.Item to='/profile' name='profile' active={activeItem === 'profile'} onClick={this.handleItemClick}>
-						Profile
-					</Menu.Item>
-		    	<Menu.Item to='/login' name='sign in' active={activeItem === 'sign in'} onClick={this.handleItemClick}>
-						Sign In
-					</Menu.Item>
-		    </Menu.Menu>
-		  </Menu>
+  		<Grid>
+				<Grid.Row only='computer'>
+					<Grid.Column>
+				  	<Menu color='blue' inverted pointing secondary>
+					    <Menu.Item name='Fantasy Fitness' />
+							<MenuItemsRouting activeItem={activeItem} handleItemClick={this.handleItemClick} />	
+						</Menu>	
+						<RoutingPaths />
+					</Grid.Column>					
+				</Grid.Row>
+			  <Grid.Row only='tablet mobile'>
+			  	<Grid.Column>
+			  		<Menu color='blue' inverted>
+			  			<Menu.Item name='Fantasy Fitness' />
+			  			<Menu.Menu position='right'>
+			  				<Menu.Item onClick={this.toggleVisibility}>
+			  					<Icon name='sidebar' />
+			  				</Menu.Item>
+			  			</Menu.Menu>
+			  		</Menu>
+		  			<Sidebar.Pushable as={Segment}>
+		  				<Sidebar as={Menu} animation='overlay' width='wide' visible={visible} vertical inverted>
+								<MenuItemsRouting activeItem={activeItem} handleItemClick={this.handleItemClick} style={{padding: '20px', fontSize: '1em'}}/>		
+		  				</Sidebar>
```

Comment:
Right now the sidebar is only rendered underneath the header and is limited to the size of the the content on the page - so if the page doesn't have content going to the bottom, the sidebar size is limited.

<img width="582" alt="screen shot 2017-11-05 at 9 51 15 am" src="https://user-images.githubusercontent.com/9142263/32415877-ec7e45e0-c20e-11e7-8b1e-af24b0016fe7.png">

Generally navigation sidebars will take up the full height of the screen

Reply:
Not sure how to handle this. Our content is what controls the height of the screen. I'm thinking I will want to add a minimum size to each of our components that displays the size of the viewport screen. That way, if there isn't enough content, it will show the navigation sidebar to the full height of the screen.